### ISSUE TITLE
No UI feedback when input paramater *.loc file is empty

### ISSUE THREAD

**Issue by peterjc** [galaxyproject/galaxy#6412]
Steps to reproduce:

1. Goto http://usegalaxy.org
2. Left hand column, search for ``blastp``
3. Open the NCBI BLAST+ blastp wrapper
4. Leave the ``Subject database / sequences`` as ``Locally installed databases``
5. Try to pick one of the locally installed databases defined in ``blastdb_p.loc``

Expected result:

Something like a disabled control with text like "No entries available"

Actual result:

Empty text box, no auto complete, no drop down menu

Source to tool:
https://github.com/peterjc/galaxy_blast/blob/master/tools/ncbi_blast_plus/ncbi_blastp_wrapper.xml
https://github.com/peterjc/galaxy_blast/blob/master/tools/ncbi_blast_plus/ncbi_macros.xml

```xml
            <when value="db">
                <param name="database" type="select" multiple="true" label="Protein BLAST database">
                    <options from_data_table="blastdb_p" />
                </param>
                <param name="histdb" type="hidden" value="" />
                <param name="subject" type="hidden" value="" />
            </when>
```

**Comment by guerler** [comment]
Thanks for reporting this. This should be fixed in dev with #6251. Will re-open if not.

**Comment by peterjc** [comment]
Thanks 'Sam, you're right - an exact duplicate of issue #6249.
